WinUIGallery/DataModel/ControlInfoData.json

```diff
@@ -2118,6 +2118,21 @@
               "Uri": "https://github.com/MicrosoftEdge/WebView2Samples"
             }
           ]
+        },
+        {
+          "UniqueId": "MapControl",
+          "Title": "MapControl",
```

Comment:
Nit: looks like all other items are sorted alphabetically - maybe this one should be as well?

![image](https://github.com/microsoft/WinUI-Gallery/assets/9866362/fa0a73a3-11da-4f65-9cb3-b99cbd08b7eb)